logstash-core/src/main/java/org/logstash/plugins/outputs/Stdout.java

```diff
@@ -20,7 +20,7 @@
 public class Stdout implements Output {
 
     public static final PluginConfigSpec<Codec> CODEC_CONFIG =
-            PluginConfigSpec.codecSetting("codec", "java-line");
+            PluginConfigSpec.codecSetting("codec", "java_line");
```

Comment:
is there some test that could be written to fail without this change?

Reply:
The codec name is resolved at runtime so a test for this would require Logstash to be running. I've been thinking about how to set up integration tests for Java tests for Java plugins and other Java code, but I haven't come up with anything simple, yet.